converter-method-kata-solutions/src/test/java/org/eclipse/collections/convertermethodkata/RichIterableToImmutableCollectionsTest.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2020 The Bank of New York Mellon.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.convertermethodkata;
+
+import java.util.Comparator;
+
+import org.eclipse.collections.api.bag.ImmutableBag;
+import org.eclipse.collections.api.bag.MutableBag;
+import org.eclipse.collections.api.bag.sorted.ImmutableSortedBag;
+import org.eclipse.collections.api.bag.sorted.MutableSortedBag;
+import org.eclipse.collections.api.factory.Bags;
+import org.eclipse.collections.api.factory.Lists;
+import org.eclipse.collections.api.factory.Maps;
+import org.eclipse.collections.api.factory.Sets;
+import org.eclipse.collections.api.factory.SortedBags;
+import org.eclipse.collections.api.factory.SortedMaps;
+import org.eclipse.collections.api.factory.SortedSets;
+import org.eclipse.collections.api.factory.Stacks;
+import org.eclipse.collections.api.list.ImmutableList;
+import org.eclipse.collections.api.list.MutableList;
+import org.eclipse.collections.api.map.ImmutableMap;
+import org.eclipse.collections.api.map.MutableMap;
+import org.eclipse.collections.api.map.sorted.ImmutableSortedMap;
+import org.eclipse.collections.api.map.sorted.MutableSortedMap;
+import org.eclipse.collections.api.set.ImmutableSet;
+import org.eclipse.collections.api.set.MutableSet;
+import org.eclipse.collections.api.set.sorted.ImmutableSortedSet;
+import org.eclipse.collections.api.set.sorted.MutableSortedSet;
+import org.eclipse.collections.api.stack.ImmutableStack;
+import org.eclipse.collections.api.stack.MutableStack;
+import org.eclipse.collections.impl.list.Interval;
+import org.eclipse.collections.impl.test.Verify;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Convert the types in the following tests to the missing type using APIs on the RichIterable interface.
+ */
+public class RichIterableToImmutableCollectionsTest
+{
+    public static final Person MARY_SMITH = new Person("Mary", "Smith", 25);
+    public static final Person TED_FIELDS = new Person("Ted", "Fields", 30);
+    public static final Person SALLY_GOLD = new Person("Sally", "Gold", 42);
+    private final ImmutableList<Person> people = Lists.immutable.with(MARY_SMITH, TED_FIELDS, SALLY_GOLD);
+
+    @Test
+    public void toList()
+    {
+        Interval interval = Interval.oneTo(5);
+        // Convert interval to a MutableList<Integer>
```

Comment:
Hi @donraab, this comment should say ImmutableList. Same pattern in the rest of the PR.

Reply:
Good catch. Thanks!